db/migrate/20170704102536_use_deleted_in_containers_tables.rb

```diff
@@ -0,0 +1,55 @@
+class UseDeletedInContainersTables < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+  end
+
+  class ContainerGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ContainerImage < ActiveRecord::Base
+  end
+
+  class ContainerProject < ActiveRecord::Base
+  end
+
+  class ContainerNode < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Container < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def disconnect_to_soft_delete(model)
+    model.where.not(:deleted_on => nil).update_all("ems_id = old_ems_id")
```

Comment:
@zeari @zakiva @simon3z 
Is there any scenario where a record could have `ems_id` set and `old_ems_id == nil`, but `deleted_on` set?
Then this would make a bad situation worse, setting `ems_id = old_ems_id = nil`.

It "shouldn't happen", I just remember we saw `ems_id == old_ems_id == nil` in a customer's DB, and suspected it might have resulted from double disconnect overwriting old_ems_id, so trying to think along similar lines...

Same question for `soft_delete_to_disconnect`.